src/utils/file.js

```diff
@@ -83,3 +83,16 @@ export function createIndexFile(folderName) {
     loadTemplate('./../../templates/js/index.js')
   );
 }
+
+export function createReadMe(folderName, configs) {
+  winston.log('verbose', 'createReadMe', { projectName: configs.projectConfigurations.projectName, description: configs.projectConfigurations.description });
+
+  const readmeContents = `# ${configs.projectConfigurations.projectName}\n`
```

Comment:
Put another newline after the project name (`\n\n`), and remove the paragraph element tags (`<p>...</p>`). In markdown, you don't need them. Non-header lines become paragraphs automatically. Also, change Tyr-CLI to `[Tyr-CLI](https://github.com/hammer-io/tyr)` so it hyperlinks to the tyr GitHub page.